CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
+## [0.2.2] - 2023-01-25
```

Comment:
This should probably be unreleased as this version is not released yet

Reply:
Fixed.

Reply:
I don't see the fix in the PR

Reply:
Forgot to push, my bad.